shared/ui/src/commonMain/kotlin/com/adammcneilly/pocketleague/shared/ui/stats/CoreStatsComparisonCard.kt

```diff
@@ -31,32 +32,52 @@ fun CoreStatsComparisonCard(
                 .fillMaxWidth()
                 .padding(16.dp),
         ) {
-            Text(
-                text = "Goals",
+            StatLine(
```

Comment:
Instead of a long list of hardcoded stat lines, we can have the stats display model expose a list of stats, and just for loop through that.